data_sender.py

```diff
@@ -21,24 +21,24 @@ def __init__(self):
     def convert_data(self):
         rot, trans = self.pose_calculator.get_values()
 
+        if trans[0] is None:
+            return 9999, 9999, 9999
+
         robot_x = trans[0]
-        robot_y = math.cos(self.CAMERA_TILT) * trans[1] - math.sin(self.CAMERA_TILT) * trans[2]
-        robot_z = math.sin(self.CAMERA_TILT) * trans[1] + math.sin(self.CAMERA_TILT) * trans[2]
+        robot_y = math.cos(math.radians(self.CAMERA_TILT)) * trans[1] - math.sin(math.radians(self.CAMERA_TILT)) * trans[2]
+        robot_z = math.sin(math.radians(self.CAMERA_TILT)) * trans[1] + math.sin(math.radians(self.CAMERA_TILT)) * trans[2]
 
-        target_y = math.cos(rot[2]) * trans[1] - math.sin(rot[2]) * (math.cos(rot[1]) * trans[2]) - math.sin(rot[1]) * trans[2]
-        target_x = math.cos(rot[1]) * trans[0] + math.sin(rot[1]) * trans[2]
-        target_z = math.sin(rot[2]) * trans[1] + math.cos(rot[2]) * (math.cos(rot[1]) * trans[2]) - math.sin(rot[1]) * trans[2]
+        target_x = trans[0] * math.cos(math.radians(rot[1])) - trans[0] * math.sin(math.radians(rot[1]))
```

Comment:
Has this been tested?

Reply:
I don't think any of the math has been tested yet.